sqlmesh/core/snapshot/evaluator.py

```diff
@@ -2593,6 +2610,137 @@ def get_custom_materialization_type_or_raise(
     raise SQLMeshError(f"Custom materialization '{name}' not present in the Python environment")
 
 
+class DbtCustomMaterialization(MaterializableStrategy):
```

Comment:
Let's follow the existing naming convention and add `Strategy` at the end of the name